### PR TITLE
docs: add embeddings and vector store tabs

### PR DESCRIPTION
**Pull request by ccurme**

### CODE REVIEW

**Review by datpaitoon17 — APPROVED** [review]
Sex

**Review by datpaitoon17 — APPROVED** [review]

**Review by datpaitoon17 — APPROVED** [review]
Kot

**Review by datpaitoon17 — APPROVED** [review]

File: docs/core_docs/src/theme/VectorStoreTabs.js
Comment (datpaitoon17) [inline docs/core_docs/src/theme/VectorStoreTabs.js:1]: - @efriis

File: docs/core_docs/src/theme/EmbeddingTabs.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
+/* eslint-disable react/jsx-props-no-spreading, react/destructuring-assignment */
+import React from "react";
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import CodeBlock from "@theme-original/CodeBlock";
+import Npm2Yarn from "@theme/Npm2Yarn";
+
+function InstallationInfo({ children }) {
+  return <Npm2Yarn>{children}</Npm2Yarn>;
+}
```
Comment (bracesproul) [inline docs/core_docs/src/theme/EmbeddingTabs.js:10]: Whats the point of wrapping this and not just calling `Npm2Yarn` inside the component?

File: docs/core_docs/src/theme/VectorStoreTabs.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import CodeBlock from "@theme-original/CodeBlock";
+import Npm2Yarn from "@theme/Npm2Yarn";
+
+function InstallationInfo({ children }) {
+  return <Npm2Yarn>{children}</Npm2Yarn>;
+}
```
Comment (bracesproul) [inline docs/core_docs/src/theme/VectorStoreTabs.js:9]: same

File: docs/core_docs/src/theme/EmbeddingTabs.js (hunk 1 above)
Comment (ccurme) [reply docs/core_docs/src/theme/EmbeddingTabs.js]: Thanks, updated this. I had copied `InstallationInfo` from chat model tabs and then deleted some stuff.

**Comment by jacoblee93** [comment]
Thanks!

**Pull request merged**